.github/container/Dockerfile.maxtext

```diff
@@ -0,0 +1,37 @@
+# syntax=docker/dockerfile:1-labs
+
+ARG BASE_IMAGE=ghcr.io/nvidia/jax:mealkit
+ARG REPO_MAXTEXT=https://github.com/google/maxtext.git
+ARG REF_MAXTEXT=main
+ARG SRC_PATH_MAXTEXT=/opt/maxtext
+
+###############################################################################
+## Download source and add auxiliary scripts
+###############################################################################
+
+FROM ${BASE_IMAGE} as mealkit
+
+ARG REPO_MAXTEXT
+ARG REF_MAXTEXT
+ARG SRC_PATH_MAXTEXT
+
+RUN <<"EOF" bash -ex
```

Comment:
Is there a particular reason to keep this out of the central manifest.yaml? Putting it here means that your `latest_verified_commit` won't be bumped by our CI so JAX will move ahead and maxtext will stay where it is.